### PR TITLE
Add gobject introspection run dep

### PR DESCRIPTION
**Pull request by mrakitin**
Prompted by https://github.com/bluesky/hklpy/pull/211.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mrakitin** [comment]
Discussion in https://github.com/bluesky/hklpy/pull/211#discussion_r763168629 is still going on to see if we actually need it. Converted to draft until we come up with the decision.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -37,6 +37,7 @@ requirements:
     - python
   run:
     - python
+    - gobject-introspection
```
Comment (prjemian) [inline recipe/meta.yaml:40]: This change seems redundant.  Let's try building the latest tag (3001) in #13.

**Review by prjemian — CHANGES_REQUESTED** [review]
Suggest closing this PR to move ahead with #13

**Comment by prjemian** [comment]
@mrakitin Ok if I close this PR?

**Comment by mrakitin** [comment]
@prjemian, yes, let's close it and iterate in a newer one.

**Pull request closed without merging**